src/main/resources/postinst.ftl

```diff
@@ -1,9 +1,8 @@
-USER_HOME=`echo ~${config.user}`
 
 case "$1" in
         configure)
-        chown -R ${config.user}:${config.group} $USER_HOME/${config.artifactId}
-        chmod -R 750  $USER_HOME/${config.artifactId}
+        chown -R ${config.user}:${config.group} "/${config.installDir}""
```

Comment:
I guess should be single " instead of ""

Reply:
You're right - this was the AI of the IDE